tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/api/IActionBarsTest.java

```diff
@@ -150,11 +150,13 @@ public void testSetGlobalActionHandler() throws Throwable {
 			MockAction cut = new MockAction();
 			MockAction copy = new MockAction();
 			MockAction undo = new MockAction();
+			MockAction quit = new MockAction();
```

Comment:
You didn't change anything for the "Quit" action. Why is this change in the test needed?

Reply:
Because the test checks both ways, the Retarget Action and the E4 Action. Quit was the only action I found which was available in the test environment and still a Retarget Action.